crates/holochain/src/conductor/cell.rs

```diff
@@ -911,3 +911,9 @@ impl Cell {
         &self.queue_triggers
     }
 }
+
+impl std::fmt::Debug for Cell {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_fmt(format_args!("Cell(id = {:?})", self.id()))
```

Comment:
If you use the helper fns you get the `{:?}` vs `{:#?}` formatting for free:

```suggestion
        f.debug_struct("Cell").field("id", &self.id()).finish()
```